src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/integration/wiremock/ActivitiesAppointmentsApiMockServer.kt

```diff
@@ -119,7 +120,7 @@ class ActivitiesAppointmentsApiMockServer : MockServer(8089) {
         appointmentSeriesId = 1,
         appointmentName = "appointment name",
         attendees = listOf(AppointmentAttendeeSearchResult(1, prisonerNumber, 1)),
-        category = AppointmentCategorySummary("VLB", "video link booking"),
+        category = AppointmentCategorySummary(locationType, "video link booking"),
         inCell = false,
```

Comment:
Is this right? LocationType going into category?

Reply:
I believe so yes.  Bare in mind I have just called it this in the test so it reads as location type when calling.